docs/adr/adr-006-befp.md

```diff
@@ -0,0 +1,110 @@
+# ADR #006: Celestia-Node Bad encoding fraud proof(BEFP)
+
+## Changelog
+
+- 2022.03.03 - init commit
+- 2022.03.08 - added pub-sub
+
+## Authors
+
+@vgonkivs @Bidon15 @adlerjohn @Wondertan
+
+## Context
+
+In case when node receive `ErrByzantineRow`/`ErrByzantineCol` from the [rsmt2d](https://github.com/celestiaorg/rsmt2d) library it should generate a fraud proof that gets broadcasted to light clients to inform them that block could be malicious.
```

Comment:
I think we can be more specific here on what exact type of node will do what for whom. Please consider this suggestion below 👇 

```suggestion
In case when a Full Node receives `ErrByzantineRow`/`ErrByzantineCol` from the [rsmt2d](https://github.com/celestiaorg/rsmt2d) library, it generates a fraud proof.  After generation of it(fraud proof), a Full Node is broadcasting the fraud proof to Light Nodes, so they(Light Nodes) are notified that a block could be malicious.
```